config/config.json

```diff
@@ -1,11 +1,23 @@
 {
   "production": {
-    "apiBase": "https://api.XXXXX.com/"
+    "apiEndpoints": {
+      "routes": "http://admin.XXXXX.com/wp-json/lean/v2/routes",
+      "static": "http://admin.XXXXX.com/wp-json/lean/v2/static",
+      "adminBar": "http://admin.XXXXX.com/wp-json/lean/v2/admin-bar"
+    }
   },
   "staging": {
-    "apiBase": "http://api.XXXXX.moxie-staging.com/"
+    "apiEndpoints": {
+      "routes": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/routes",
+      "static": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/static",
+      "adminBar": "http://admin.XXXXX.moxie-staging.com/wp-json/lean/v2/admin-bar"
+    }
   },
   "dev": {
-    "apiBase": "http://localhost:3000/"
+    "apiEndpoints": {
```

Comment:
This does not work with the json-server anymore. Are we going to deprecate it?

Also it would be nice to have the api base defined in only one place, i.e.:

```
{
     "staging": {
          "apiBase": "http://api.XXXXX.moxie-staging.com/wp-json/lean/v2/"
          "apiEndpoints": {
               "routes": "{STAGING_API_BASE}routes",
               "static": "{STAGING_API_BASE}static",
               "adminBar": "{STAGING_API_BASE}admin-bar"
          }
     },
}
```

And replace `{STAGING_API_BASE}` with the value of staging.apiBase in the configFunc of the gulpfile.